.github/workflows/maestro_release.yaml

```diff
@@ -0,0 +1,32 @@
+name: Maestro Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install uv and activate the environment
+        uses: astral-sh/setup-uv@v6
+      - name: Build Maestro
+        shell: bash
+        run: uv build
+      - name: GitHub Release
+        uses: softprops/action-gh-release@v2
+        if: github.ref_type == 'tag'
+        with:
+          files: dist/*
+          generate_release_notes: true
+#      - uses: actions/upload-artifact@v4
```

Comment:
Why these commented lines? Get rid of them. It's in git log if you need to ever go back.